etc/DependencyInstaller.sh

```diff
@@ -309,6 +323,12 @@ while [ "$#" -gt 0 ]; do
         -dev|-development)
             option="dev"
             ;;
+        -local)
+            export PREFIX="$HOME/.local"
+            ;;
+        -prefix=*)
+            export PREFIX=`echo $1 | sed -e 's/^[^=]*=//g'`
```

Comment:
```suggestion
            export PREFIX="$(echo $1 | sed -e 's/^[^=]*=//g')"
```
surround with quotes to consider whitespace? shell can be tricky with spaces, but at least here it should work.